generator/scripts/generateInputRange.py

```diff
@@ -0,0 +1,97 @@
+import openpyxl  # Read data from Excel
+
+
+# Function to replace single quotes and stringify text
+def replaces_quotes_and_stringify(text):
+    if text is not None:
+        return str(text).replace("'", "\\'")  # Replace single quotes
+    return None
+
+
+# Function to generate inputRange.tsx
+def generate_input_range(input_file: str, output_file: str):
+    try:
+        # Load Excel file
+        workbook = openpyxl.load_workbook(input_file, data_only=True)
+        sheet = workbook["InputRange"]  # Get InputRange sheet
+
+        # Get headers from the first row
+        headers = [cell.value for cell in list(sheet.rows)[0]]
+
+        # Generate TypeScript code
+        ts_code: str = ""  # TypeScript code to be written to file
+        indentation: str = "    "  # 4-space indentation
+
+        # Add imports
+        # ts_code += "import { Labels } from '../types/inputTypes';\n\n"
+
+        # Iterate through each row in the sheet, starting from the second row
+        for row in list(sheet.rows)[1:]:
+            # Create a dictionary from the row values and headers
+            row_dict = dict(zip(headers, (cell.value for cell in row)))
+
+            # Get values from the row dictionary
+            input_range_name = row_dict["inputRangeName"]
+            label_fr_min = replaces_quotes_and_stringify(row_dict["labelFrMin"])
```

Comment:
Add a TODO to support more languages. Ideally, to prepare for this future support, rename to labelMin_fr/labelMin_en, such that the string after the `_` character is the locale name.

Reply:
Great idea.